pkg/cluster/k8sres.go

```diff
@@ -221,7 +222,22 @@ PatroniInitDBParams:
 	config.PgLocalConfiguration = make(map[string]interface{})
 	config.PgLocalConfiguration[patroniPGBinariesParameterName] = fmt.Sprintf(pgBinariesLocationTemplate, pg.PgVersion)
 	if len(pg.Parameters) > 0 {
-		config.PgLocalConfiguration[patroniPGParametersParameterName] = pg.Parameters
+		localParameters := make(map[string]string)
```

Comment:
bootstrap == params used during the startup of Spilo, and local == everything used after bootstrap ?